src/com/mellanox/cicd/Matrix.groovy

```diff
@@ -258,7 +258,9 @@ def runK8(image, branchName, config, axis) {
         node(POD_LABEL) {
             stage (branchName) {
                 container(cname) {
-                    runSteps(image, config)
+                    catchError(buildResult: 'FAILURE', stageResult: 'FAILURE') {
```

Comment:
yaml has ```failFast: false``` param for it
why it is not enough?

```
# Fail job is one of the steps fails or continue
failFast: false
```

Reply:
Wow, great! Sorry, haven't seen it before.